### DISCUSSION TITLE
Is it possible to generate a video with burn-in subs ?

### DISCUSSION THREAD

**Discussion by nounours18200** [URUWorks/TeroSubtitler#194] [General]
Hello,
When the subs are completed, is it possible to generate a video with the burn-in subtitles ? (if yes, how ?).
Thank you

**Comment by chenlung** [comment]
I suspect this would not be within the remit of what is a Subtitle Editor (a video editor/encoder would be more appropriate, such as [Shutter Encoder](https://github.com/paulpacifico/shutter-encoder)).

**Reply by nounours18200** [reply]
I sometimes use Shutter Encoder, but I have not yet used it for this purpose: I do not see how to import a subtitle file, in order to burn-in it in a video file (???).
I have launched a burn-in process with Handbrake (not yet completed), that is supposed to do this.
SubtitlesEdit v4 does it perfectly, but it compresses the video, and it should be fixed for the process without compression....
-----------------
Back to TeroSubtitler: it seems an excellent work, but the AudioWave form is not generated on my PC. It looks like a promising tool.... congrats.

**Comment by chenlung** [comment]
Please use a normal font.

Shutter Encoder is not a subtitle editor, but it has more encoding options as a dedicated tool. Although it supports WebVTT and SRT, more formats would be useful.

I think Subtitle Edit can indeed produce a video with burned-in subtitles (I don't know if it's just MP4). However, I'd rather more resources were invested in improving the subtitling aspect of Tero instead of a feature that another tool can and is more suited to.

Could you elaborate on the waveform issue, please? Are you sure that FFmpeg is installed correctly? What OS and version are you using?

**Comment by dadoprom** [comment]
Just use ffmpeg for that purpose. Here I attach script I use for that purpose:
[burn subs script.zip](https://github.com/URUWorks/TeroSubtitler/files/13239654/burn.subs.script.zip)

**Comment by URUWorks** [comment]
you can try this version:

https://github.com/URUWorks/additional-files/raw/main/terosubtitler_testbuild_win64.zip
